lib/submariner_prepare/downstream_prepare.sh

```diff
@@ -114,6 +114,26 @@ function create_catalog_source() {
     done
 }
 
+function update_catalog_source() {
+    INFO "Increase the submariner version"
+    local submariner_version="$SUBMARINER_VERSION_INSTALL"
+    submariner_version=${submariner_version%.*}
+    submariner_version=$(bc <<< "$submariner_version+0.01")
+    submariner_version=$(printf "%.2f" "$submariner_version")
+    export SUBMARINER_VERSION_INSTALL=$submariner_version
+
+    INFO "Update catalog source on the managed clusters for submariner version - $SUBMARINER_VERSION_INSTALL"
+    for cluster in $MANAGED_CLUSTERS; do
+        get_latest_iib
+        image_source="$LATEST_IIB"
```

Comment:
You're actually making identical versions assignment here, in line 128 and in line 131.
Please, remove one of them.